cli/azd/pkg/container/dependencies.go

```diff
@@ -0,0 +1,13 @@
+package container
```

Comment:
Are you intending this package to be the IoC package moving forward? I'm convinced that we can and should use a DI framework like google/wire. Registration is cheap, injection, dependency ordering and meaningful errors are harder to achieve.

It's something that is high on my priority list to explore.

Reply:
Yes, very open to adopting a real IoC package or similar project.